src/node_sqlite.cc

```diff
@@ -168,6 +168,257 @@ inline MaybeLocal<Value> NullableSQLiteStringToValue(Isolate* isolate,
       .As<Value>();
 }
 
+class CustomAggregate {
```

Comment:
There is **a lot** of duplication between this and the `UserDefinedFunction` class. I think we should combine them.

Reply:
Yeah, I am cleaning up the code at this moment. I think I can make CustomAggregate inherit from UserDefined and save a lot of code.

Would you have any objection?

Reply:
No objection, but we probably don't even need a separate class.

Reply:
Hmm, good point.